src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -660,7 +659,7 @@ Array [
   });
 
   describe('hydrate', () => {
-    it('sets __initialSearchResults', () => {
+    it('sets _initialResults', () => {
```

Comment:
It would be better to test the consequence of setting `_initialResults` rather than setting it.

Reply:
The later tests (setting helper, and init index) already test that, right? Do you have an idea which test to write?

Reply:
I think that this test covers it: [`904bde4` (#1090)](https://github.com/algolia/vue-instantsearch/pull/1090/commits/904bde4464dd1c90c787fadf51de7937efc3f9e6#diff-7f49c18a452238e67a534ec6a7197429b8cdc0da21ed98ea38e1ec85ece88201R923)

We may not need this test altogether if the behavior is tested at a higher level elsewhere then.